wincode/src/serde.rs

```diff
@@ -227,6 +227,44 @@ where
     T::deserialize(src)
 }
 
+/// Deserialize a mutable reference to a type from the given bytes.
+///
+/// This allows mutating the serialized data in place.
+///
+/// Only available for types that are [`ZeroCopy`](crate::ZeroCopy).
```

Comment:
This is not the case anymore, right?

Reply:
Yes, that is a good idea. Maybe we can have an example of a "PartialZeroCopy" struct as well.